src/Cultuurkuur/data/regions.json

```diff
@@ -0,0 +1,5710 @@
+[
+    {
+        "label": "Brussels Hoofdstedelijk Gewest",
+        "value": "nis-01000",
+        "children": [
+            {
+                "label": "Regio Brussel",
+                "value": "reg-brussel",
+                "children": [
+                    {
+                        "label": "Anderlecht + deelgemeenten",
+                        "value": "nis-21001"
```

Comment:
This `nis-21001` is wrong and should be `cultuurkuur_werkingsregio_nis-21001` otherwise the wrong label will be added.